web/client/components/widgets/enhancers/wpsChart.js

```diff
@@ -6,12 +6,26 @@
   * LICENSE file in the root directory of this source tree.
   */
 const {compose, withProps} = require('recompose');
+const {isObject} = require('lodash');
 const wpsAggregate = require('../../../observables/wps/aggregate');
 const propsStreamFactory = require('../../misc/enhancers/propsStreamFactory');
 const Rx = require('rxjs');
 const wpsAggregateToChartData = ({AggregationResults = [], GroupByAttributes = [], AggregationAttribute, AggregationFunctions} = {}) =>
-    AggregationResults.map( (res) => ({
-        ...GroupByAttributes.reduce( (a, p, i) => ({...a, [p]: res[i]}), {}),
+    AggregationResults.map((res) => ({
+        ...GroupByAttributes.reduce((a, p, i) => {
+            let value = res[i];
+            if (isObject(value)) {
+                if (value.time) {
+                    value = (new Date(value.time)).toISOString();
+                } else {
+                    value = JSON.stringify(value.time);
```

Comment:
sorry, maybe here you should throw the error, so the error is shown on chart. 
My TODO in the comment didn't make sense. For instance here the format of the server is not expected